installers/binstall/src/install.rs

```diff
@@ -18,16 +18,6 @@ impl Installer {
     pub fn install(&self) -> Result<Option<PathBuf>, InstallerError> {
         let install_path = self.do_install()?;
 
-        // On Windows we likely popped up a console for the installation. If we were
-        // to exit here immediately then the user wouldn't see any error that
-        // happened above or any successful message. Let's wait for them to say
-        // they've read everything and then continue.
-        if cfg!(windows) {
-            tracing::info!("Press enter to close this window...");
-            let mut line = String::new();
-            drop(io::stdin().read_line(&mut line));
-        }
```

Comment:
What's the point of getting rid of this?

Reply:
this was taken from `wasm-pack` which provides an `.exe` download, which we don't . since folks will be running this in powershell rather than the automatic `cmd.exe` this was primarily focused on, displaying this message is not particularly accurate anymore!